src/python/platforms/android/gestures.py

```diff
@@ -22,28 +22,26 @@
 MONKEY_THROTTLE_DELAY = 100
 
 # Maximum number of monkey events per testcase.
-NUM_MONKEY_EVENTS = 50
+NUM_MONKEY_EVENTS = 25
```

Comment:
Won't this cause issues with reproducibility for existing tests? Or do we not expect to have any relevant ones open at the moment?

Reply:
Nothing interesting is open atm, so this is fine.